content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -266,6 +275,16 @@ an `ingressClassName` specified. You can resolve this by ensuring that at most 1
 IngressClass is marked as default in your cluster.
 {{< /caution >}}
 
+There are some ingress controllers, that work without the definition of a default
+`IngressClass`, even on newer versions of Kubernetes and the controllers. For
+example, the Ingress-NGINX controller can be configured with a
+[flag](https://kubernetes.github.io/ingress-nginx/#what-is-the-flag-watch-ingress-without-class)
+`--watch-ingress-without-class`. Although this works for now, it is highly
```

Comment:
Is the "Although it works for now" statement accurate? I wanted to highlight, that we should use the `IngressClass`, even if it still works without it in some cases. The NGINX documentation/blog posts are also not 100% clear about that. It works without, but they recommend to use the `IngressClass` 😏

Reply:
I _think_ there's a way to set up a cluster so that there's no IngressClass at all, and there are working Ingresses, and that cluster runs v1.22 Kubernetes.

I'd try to avoid “for now” in tech docs; the ideal is that the content is [timeless](https://developers.google.com/style/timeless-documentation) (not our guide, but we borrowed the idea from there).

Reply:
Yes, according to the Ingress-NGINX docs you sent, you can specify that flag, and then you don't need the IngressClass. 

Agreed on the timeless. I'll adopt the sentence.